examples/soroban/nft/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+#![no_std]
+use loam_sdk::soroban_contract;
+use loam_subcontract_core::{admin::Admin, Core};
+use loam_soroban_sdk::{Address, Bytes};
```

Comment:
I needed to include `use loam_soroban_sdk::{Address, Bytes};` to make the code compile otherwise I was seeing this error:
```
error[E0412]: cannot find type `Bytes` in this scope
  --> examples/soroban/nft/src/lib.rs:28:1
  |
28 | soroban_contract!();
  | ^^^^^^^^^^^^^^^^^^^ not found in this scope
```

Not sure if there is a better way to handle this, as I didn't see other examples needing to do this 🤔

Reply:
The issue here was that the methods from the trait are copy/pasted here. So you need to fully qualify the types to avoid needed to import them again.

```rust
impl SorobanContract {
    //...
    // Get the NFT with the given ID
    fn get_nft( id: u32) -> Option<loam_sdk::soroban_sdk::Bytes> {
        Contract::lazy_get().unwrap().get_nft(id)
    }
}
```

From the outside the trait doesn't have any `self` references. So this code is generated and to invoke the method bound to `Contract` via the `NonFungible` trait.

Reply:
Ohhh, that is helpful, thanks. I may need to think on this a bit more, but I think that makes sense 👌